apps/user-office-backend/src/datasources/stfc/StfcProposalDataSource.ts

```diff
@@ -57,7 +57,9 @@ export default class StfcProposalDataSource extends PostgresProposalDataSource {
         if (filter?.text) {
           query
             .where('title', 'ilike', `%${filter.text}%`)
-            .orWhere('abstract', 'ilike', `%${filter.text}%`);
+            .orWhere('proposal_id', 'ilike', `%${filter.text}%`)
```

Comment:
Maybe we can save the odd request by checking that filter.text is not an empty string

Reply:
I did a little check and if empty it will not pass the if statement check we must be covered.

Reply:
Ok sure. Don't pick up the other one though as it's very very low priority :)